Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -33,6 +34,13 @@ class MainTabBarViewController: UITabBarController {
             object: nil)
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        guard appearsFirstTime else { return }
+        appearsFirstTime = false
```

Comment:
Is it needed because of the lock screen?

Reply:
yes, because otherwise rate app appears on lock screen